src/trans.c

```diff
@@ -2022,6 +2020,8 @@ Obj FuncON_KERNEL_ANTI_ACTION(Obj self, Obj ker, Obj f, Obj n){
     if(len>=deg){
       pttmp=ResizeInitTmpTrans(len);
       ptf2=ADDR_TRANS2(f);
+      out=NEW_PLIST(T_PLIST_CYC+IMMUTABLE, len);
+      SET_LEN_PLIST(out, len);
     
```

Comment:
If a garbage collection occurs when calling `NEW_PLIST`, the pointer to `ptf2` will not be valid.